src/Features/Core/Portable/Completion/CompletionService.cs

```diff
@@ -171,24 +171,6 @@ public virtual TextSpan GetDefaultCompletionListSpan(SourceText text, int caretP
             return Task.FromResult(CompletionChange.Create(new TextChange(item.Span, item.DisplayText)));
         }
 
-        /// <summary>
-        /// Preferred overload of <see cref="GetChangeAsync(Document, CompletionItem, char?,
-        /// CancellationToken)"/>.
-        ///
-        /// This overload is passed the value of <see cref="CompletionContext.CompletionListSpan"/>
-        /// which should be used to determine what span should be updated in the original <paramref
-        /// name="document"/> passed in.
-        /// </summary>
-        internal virtual Task<CompletionChange> GetChangeAsync(
```

Comment:
You'll need to remove the exposed member in the OmniSharp external access as well.

Reply:
sgtm